pages.es/common/upt.md

```diff
@@ -0,0 +1,38 @@
+# upt
+
+> Interfaz unificada para gestionar paquetes en varios sistemas operativos, tales como Windows, diversas distribuciones de Linux, macOS, FreeBSD e incluso Haiku.
+> Requiere que el gestor de paquetes nativo del sistema operativo esté instalado.
```

Comment:
```suggestion
> Requiere que el gestor de paquetes nativo del sistema operativo este instalado.
```

Reply:
In this case, it is a verb. So it should be with a "tilde"